infer-web.py

```diff
@@ -655,26 +655,26 @@ def change_sr2(sr2, if_f0_3, version19):
     path_str = "" if version19 == "v1" else "_v2"
     f0_str = "f0" if if_f0_3 else ""
     if_pretrained_generator_exist = os.access(
-        "pretrained%s/%sG%s.pth" % (path_str, f0_str, sr2), os.F_OK
+        "%s%s/%sG%s.pth" % (pretrained_pth_root, path_str, f0_str, sr2), os.F_OK
```

Comment:
I dont think it will not work.. AFAIK os.access cannot get external urls.

Reply:
I don't think this counts as an external link.
IIUC, anything with the path `/something` is on the filesystem (as far as the operating system is concerned).